postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -431,7 +439,15 @@ def _collect_plan_for_statement(self, row):
                 },
                 "db": {
                     "instance": row.get('datname', None),
-                    "plan": {"definition": obfuscated_plan, "cost": plan_cost, "signature": plan_signature},
+                    "plan": {
+                        "definition": obfuscated_plan,
+                        "cost": plan_cost,
+                        "signature": plan_signature,
+                        "collection_error": {
+                            "code": explain_err_code,
+                            "reason": err,
```

Comment:
Maybe `message` would be a clearer key here? "reason" is a bit close to "code" so it may be confusing.